### ISSUE TITLE
Authentification for mail alerting

### ISSUE THREAD

**Issue by FafnerKeyZee** [CIRCL/AIL-framework#213]
Hey,

I got some troubles to send alerts via mail. I'm using STARTTLS on my mail server, so I had to patch:
```
        if sender_pw is not None:
            smtp_server = smtplib.SMTP_SSL(sender_host, sender_port)
            smtp_server.ehlo()
            smtp_server.login(sender, sender_pw)
        else:
            smtp_server = smtplib.SMTP(sender_host, sender_port)
```
into:
```
        if sender_pw is not None:
            smtp_server = smtplib.SMTP(sender_host, sender_port)
            smtp_server.ehlo()
            smtp_server.starttls()
            smtp_server.ehlo()
            smtp_server.login(sender, sender_pw)
        else:
            smtp_server = smtplib.SMTP(sender_host, sender_port)
```

I think that it is better to add a new option into config file. What do you think about that ?

**Comment by adulau** [comment]
Good idea! We should have an option for TLS/ESMTP enabled server. @mokaddem @Terrtia what do you think? or fallback if the server is not supporting TLS?

**Comment by Terrtia** [comment]
should be fix with fa38434e04a9296addc8d20a9ae6b67da4b619d7

Fallback if the server is not supporting TLS

can you please confirm ?

**Comment by FafnerKeyZee** [comment]
It's working :)

Thanks for the modification.
